modules/before-updating-web-console.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: CONCEPT
+[id="before-updating-web-console{context}"]
+= Before updating using the web console
+
+Before updating using the web console, consider the following:
+
+* Your etcd backup is recent enough in case your update fails and you must restore your cluster to a previous state.
+
+* You might experience an interruption during the update process if you run an Operator or you have configured any application with the pod disruption budget. If `minAvailable` is set to 1 in `PodDisruptionBudget`, the nodes are drained to apply pending machine configs which might block the eviction process. If several nodes are rebooted, all the pods might run on only one node, and the `PodDisruptionBudget` field can prevent the node drain.
```

Comment:
Also I do not agree with the line `You might experience an interruption during the update process if you run an Operator or you have configured any application with the pod disruption budget` . I am fine with removing this line altogether.

Reply:
Trevor just created [OSDOCS-7661](https://issues.redhat.com/browse/OSDOCS-7661) to drop this item from this list entirely, would you be willing to wait until I start that ticket to make any changes to this item?

Reply:
Resolving this so I dont accidentally capture this.

Reply:
@skopacz1, @LalatenduMohanty and I discussed this. I have removed the sentence requested in this PR, but feel free to edit in however you see fit when you fix the ticket you mentioned.